infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/CoreTypes.java

```diff
@@ -38,6 +39,15 @@ public class CoreTypes {
           .format("byte")
           .build();
 
+  public static final StringValueTypeDefinition<Bytes20> BYTES20_TYPE =
```

Comment:
Do we actually want bytes20 or eth1 address here?

Reply:
probably at some stage an eth1 address is my guess, but it's pretty built into the objects at the moment that it's a bytes20... probably a bit of a cleanup, bigger than the scope of this migration